README.md

```diff
@@ -3,3 +3,91 @@
+  🤖 [Progress](https://www.hyperui.dev/components/application-ui/progress)  
+  🤖 [Radio Groups](https://www.hyperui.dev/components/application-ui/radio-groups)  
+  🤖 [Selects](https://www.hyperui.dev/components/application-ui/selects)  
+  🤖 [Side Menu](https://www.hyperui.dev/components/application-ui/side-menu)  
+  🤖 [Stats](https://www.hyperui.dev/components/application-ui/stats)  
+  🤖 [Steps](https://www.hyperui.dev/components/application-ui/steps)  
+  🤖 [Tables](https://www.hyperui.dev/components/application-ui/tables)  
+  🤖 [Tabs](https://www.hyperui.dev/components/application-ui/tabs)  
+  🤖 [Textareas](https://www.hyperui.dev/components/application-ui/textareas)  
+  🤖 [Toggles](https://www.hyperui.dev/components/application-ui/toggles)  
+  🤖 [Vertical Menu](https://www.hyperui.dev/components/application-ui/vertical-menu)  
+  
+### Marketing
+📣 [Announcements](https://www.hyperui.dev/components/marketing/announcements)  
+📣 [Banners](https://www.hyperui.dev/components/marketing/banners)  
+📣 [Blog Cards](https://www.hyperui.dev/components/marketing/blog-cards)  
+📣 [Buttons](https://www.hyperui.dev/components/marketing/buttons)  
+📣 [Cards](https://www.hyperui.dev/components/marketing/cards)  
+📣 [CTAs](https://www.hyperui.dev/components/marketing/ctas)  
+📣 [FAQs](https://www.hyperui.dev/components/marketing/faqs)  
+📣 [Footers](https://www.hyperui.dev/components/marketing/footers)  
+📣 [Forms](https://www.hyperui.dev/components/marketing/forms)  
+📣 [Header](https://www.hyperui.dev/components/marketing/headers)  
+📣 [Popups](https://www.hyperui.dev/components/marketing/popups)  
+📣 [Pricing](https://www.hyperui.dev/components/marketing/pricings)  
+📣 [Sections](https://www.hyperui.dev/components/marketing/sections)  
+📣 [Stats](https://www.hyperui.dev/components/marketing/stats)  
+📣 [Testimonials](https://www.hyperui.dev/components/marketing/testimonials)  
+
+### eCommerce
+🛍️ [Carts](https://www.hyperui.dev/components/ecommerce/carts)  
+🛍️ [Collection Cards](https://www.hyperui.dev/components/ecommerce/collection-cards)  
+🛍️ [Collection Filters](https://www.hyperui.dev/components/ecommerce/collection-filters)  
+🛍️ [Featured Filters](https://www.hyperui.dev/components/ecommerce/featured-sections)  
+🛍️ [Product Cards](https://www.hyperui.dev/components/ecommerce/product-cards)  
+🛍️ [Product Collections](https://www.hyperui.dev/components/ecommerce/product-collections)  
+🛍️ [Quantity Inputs](https://www.hyperui.dev/components/ecommerce/quantity-inputs)  
+
+## Contributing
+HyperUI accepts contributions. For guidance on how to contribute to the project, a detailed [contributing guide](https://www.hyperui.dev/blog/how-to-contribute) is available on the HyperUI [blog](https://www.hyperui.dev/blog).
+
+## License
+This project is licensed under the MIT License - see the [LICENSE](https://github.com/markmead/hyperui/blob/main/LICENSE) file for details.
```

Comment:
I think this could be condensed to:

> HyperUI is an open source project licensed under the [MIT license](https://github.com/markmead/hyperui/blob/main/LICENSE.md).